oddbird/MetaShare
# Capture changes as unpackaged metadata.

## Cute animal pic
_Because everyone likes pictures of animals._

![goat](https://q7i2y6d5.stackpathcdn.com/wp-content/uploads/2010/02/iStock-524907632-400x329.jpg)

## Link to Trello card (if applicable)
_If this PR relates to a Trello card, don't forget to reference this PR on the card as well._

https://trello.com/c/VXUHYQEZ

## Description
_The commit messages say what you did; this should explain why and how._

## Steps to test/reproduce
_Please explain how to best reproduce the issue and/or test the changes locally._

- `./derrick build`
- `./derrick migrate`
- Create a Dev Org
- Make changes in the org (https://oddbird.slack.com/archives/CH6EN9NMA/p1576536024007000)
- "check again" on Dev Org to find new changes
- Click "Capture Task Changes"

## Impacted areas of website
_Provide URLs and/or a description of where to look._

- Capture-changes modal

## Show me
_Provide screenshots/animated gifs/videos if necessary._

![Screenshot 2020-03-04 12 55 51](https://user-images.githubusercontent.com/552316/75908199-ab55b100-5e17-11ea-9c9c-f76b8ea1e063.png)

![Screenshot 2020-03-04 12 56 03](https://user-images.githubusercontent.com/552316/75908206-abee4780-5e17-11ea-872a-3cb6487f332e.png)

![Screenshot 2020-03-04 12 56 16](https://user-images.githubusercontent.com/552316/75908208-ac86de00-5e17-11ea-8855-66276b69db64.png)

---

## Reviewer tasks:
